tests/css_attribute/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -218,7 +218,7 @@ a[id='test'] {
 }
 a[id='test'] {
 }
-a[id=func('foo')] {
+a[id=func("foo")] {
```

Comment:
These should have been single quotes, right?

Reply:
not sure  will take a look  later

Reply:
`postcss-selector-parser` throws when parsering this selector `a[id=func("foo")]` 

```text
Thrown:
Error: Unexpected "(" found.
```
so prettier will not touch it

And I've nerver seen a selector like this.

Reply:
Yeah, it’s a weird test case. I wonder where it comes from. Neither SCSS nor Less seem to accept it. Maybe we can just remove it.